tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -104,14 +107,16 @@ def remove_resource(self, name, **kwargs):
         if self.is_live and self._need_creation:
             try:
                 if "wait_timeout" in kwargs:
-                    azure_poller = self.client.resource_groups.delete(name)
+                    azure_poller = self.client.resource_groups.begin_delete(name)
```

Comment:
So were all the calls to delete failing all the time before?

Reply:
Most likely yes. Any exception was being caught so there was no way to notify that it was failing.